azure_pipeline.yaml

```diff
@@ -51,7 +51,7 @@ stages:
       - template: pipeline-templates/terraform-plan.yaml
         parameters:
           environment: 'sbox'
-          component: 'cftapps_cluster_lb'
+          component: 'cftapps_cluster_lb_frontend'
```

Comment:
This is used for the state file key so you won’t want to rename if unless you rename the state file in all envs

Reply:
I realised this and reverted the change for the module. I left the naming convention for it as it was and only added "_backend" to the new module.